libpod/networking_slirp4netns.go

```diff
@@ -484,7 +485,12 @@ func (r *Runtime) setupRootlessPortMappingViaRLK(ctr *Container, netnsPath strin
 	}
 	cfgR := bytes.NewReader(cfgJSON)
 	var stdout bytes.Buffer
-	cmd := exec.Command(fmt.Sprintf("/proc/%d/exe", os.Getpid()))
+	path, err := r.config.FindHelperBinary(rootlessport.ReexecKey, false)
+	// backwards compat, keep the reexec ability
```

Comment:
Who needs this compatibility?
If the rootlessport binary is missing, isn’t the package broken?

Reply:
Yes I will remove this. I just wanted to have this in case this would be merged into 3.4. But since we are now on 4.0 I will required the binary.